lms/djangoapps/verify_student/models.py

```diff
@@ -1046,7 +1051,14 @@ def add_status_from_checkpoints(cls, checkpoints, user, status):
             None
         """
         for checkpoint in checkpoints:
-            cls.objects.create(checkpoint=checkpoint, user=user, status=status)
+            # get 'location_id' from last entry (if it exists) and add it in
+            # new entry
+            location_id = ''
+            verification_status_list = cls.objects.filter(checkpoint=checkpoint)
```

Comment:
How is this list ordered?  Should we use `latest()` to get the most recent status?